closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -38,7 +38,10 @@
 
 #define OPENSSL_VERSION_1_0 "OpenSSL 1.0."
 #define OPENSSL_VERSION_1_1 "OpenSSL 1.1."
-/* needed for OpenSSL 1.0.2 Thread handling routines */
+/* Per new OpenSSL naming convention starting from OpenSSL3, all major versions are ABI and API compatible. */
+#define OPENSSL_VERSION_3_X "OpenSSL 3"
```

Comment:
This should end with a period so it matches `"OpenSSL 3.0"` but not `"OpenSSL 32"`.

Reply:
Add a period after 3, so it will be 
`#define OPENSSL_VERSION_3_X "OpenSSL 3."`
for now